golang/streaming_stdin.go

```diff
@@ -0,0 +1,25 @@
+// Script reads streaming input text and then prints capitalized string to stdout
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func main() {
+	scanner := bufio.NewScanner(os.Stdin)
+
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		if line == "" {
+			break
+		}
+
+		fmt.Println(strings.ToUpper(line))
```

Comment:
```suggestion
		fmt.Println(strings.ToUpper(scanner.Text()))
```

(a more succinct way to write this)